SwiftDeck/SwiftDeck/Communication/CommunicationManager.swift

```diff
@@ -0,0 +1,72 @@
+//
```

Comment:
`SwiftDeck/SwiftDeck/`? is that necessary?

Is this conceptually `swift/modules/SwiftDeck`?